app/services/patient_deduplication/strategies.rb

```diff
@@ -37,7 +37,6 @@ def duplicate_identifiers
           .joins(:patient)
           .select("identifier, array_agg(patient_id) as patient_ids")
           .where.not(identifier: "")
-          .where(identifier_type: simple_bp_passport)
```

Comment:
If we're dropping this clause, then we should add `identifier_type` to the `group` clause below for accuracy.